addons/website/static/src/snippets/s_social_media/options.js

```diff
@@ -38,6 +38,9 @@ options.registry.SocialMedia = options.Class.extend({
      * @override
      */
     async cleanForSave() {
+        if (!dbSocialValues) {
+            return;
+        }
```

Comment:
Seems fine but it could use a comment to explain how dbSocialValues can be undefined in this case (just a small comment like "In case the snippet was cloned via its parent, its options UI was not updated but its options cleanForSave will be called" (something like that).